streaming/src/event_server.cc

```diff
@@ -0,0 +1,96 @@
+#include <unordered_set>
+
+#include "event_server.h"
+namespace ray {
+namespace streaming {
+EventServer::EventServer(uint32_t event_size)
```

Comment:
Will be be better to name it as `EventService` and `event_service.h`, like `boost::asio::io_service` or java `ExecutorService`. `EventServer` looks like server in client/server.